app/services/youtube_service.rb

```diff
@@ -1,4 +1,10 @@
 class YoutubeService
+
+  def create_videos(tutorial)
```

Comment:
In a previous commit I had planned on making YoutubeResults but found this YoutubeService.